protocol/messenger_test.go

```diff
@@ -309,6 +309,27 @@ func (s *MessengerSuite) TestInit() {
 	}
```

Comment:
The linter is upset about the formatting, you can fix it by running:
```
goimports -w -local 'github.com/ethereum/go-ethereum,github.com/status-im/status-go,github.com/status-im/markdown' protocol/messenger_test.go
```

That should give you some changes to commit

Reply:
done @cammellos :)